test/gui/shared/steps/steps.py

```diff
@@ -640,3 +640,11 @@ def step(context):
         'Ser&ver Address',
     )
     waitForObject(AccountConnectionWizard.SERVER_ADDRESS_BOX)
+
+
+@Then("all tabs in the toolbar should match the default baseline")
```

Comment:
could we make the tab names as parameter? So that every can be tested by itself

Reply:
Done